### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR updates Web NFC API to reflect latest spec changes.
See https://github.com/w3c/web-nfc/blob/gh-pages/EXPLAINER.md#changes-done-after-origin-trial-ot and https://web.dev/nfc/

Disclaimer: I'm an editor of the spec.

### CODE REVIEW

**Comment by Rumyra** [comment]
The changes look perfectly fine, I'm only pausing on reviewing as this change would mean moving the existing page (which I will look into 👍 )

File: files/en-us/web/api/ndefreader/index.html [hunk 1]
```diff
@@ -24,8 +24,8 @@ <h2 id="Properties">Properties</h2>
 <dl>
  <dt>{{DOMxRef("NDEFReader.onreading")}} {{Experimental_Inline}}</dt>
  <dd>An event handler for <code>reading</code> event, that notifies about availability of a new reading.</dd>
- <dt>{{DOMxRef("NDEFReader.onerror")}} {{Experimental_Inline}}</dt>
- <dd>An event handler for <code>error</code> event which is called to notify that an error occurred during reading.</dd>
+ <dt>{{DOMxRef("NDEFReader.onreadingerror")}} {{Experimental_Inline}}</dt>
+ <dd>An event handler for <code>readingerror</code> event which is called to notify that an error occurred during reading.</dd>
```
Comment (sideshowbarker) [inline files/en-us/web/api/ndefreader/index.html:28]:
```suggestion
 <dd>An event handler for the <code>readingerror</code> event, which is fired when an error occurs during reading.</dd>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Converted to draft status so that we don’t accidentally merge this before resolution of https://github.com/mdn/content/pull/683#issuecomment-751740187.

**Comment by Rumyra** [comment]
The linked page is already correctly named: https://developer.mozilla.org/en-US/docs/Web/API/NDEFReader/onreadingerror

One assumes then that `{{domxref(NDEFReader.onreadingerror)}}` will work - I'm happy to approve @sideshowbarker

**Review by Rumyra — APPROVED** [review]

**Comment by beaufortfrancois** [comment]
When will this change be live?

**Comment by sideshowbarker** [comment]
> When will this change be live?

I guess likely not until after January 4th, when the @mdn/core-yari-dev team is back.

**Comment by peterbe** [comment]
If you're waiting for a merged PR to be in production you can partially use: https://whatsdeployed.io/s/DLi/mdn/content
But the CDN will cache content too so even if your commit has made it into the daily production build, it could still be hours until you can actually see it in developer.mozilla.org.

**Comment by sideshowbarker** [comment]
> But the CDN will cache content too

Is there any way from the user side to force cache invalidation for a particular resource? I’m guessing not, but figured it worth asking a least. 

> so even if your commit has made it into the daily production build, it could still be hours until you can actually see it in developer.mozilla.org.

It seems like it would be good if we were able to give contributors some rough idea of just how long it takes in total.

For example, there’s a certain time each day when the push to production happens, right? There’s some cron job that runs at least once a day, I guess? So maybe we can document what time that actually is.

And then along with that, we have a rough idea of how long the cache invalidations take — 4 hours, or 6 hours, or 8 hours?

So maybe in the end we could say something like, _It should take at most 32 hours (or whatever the actual number works out to) for a change you contribute to finally show up on the production site_.

**Comment by beaufortfrancois** [comment]
Is it the same for mdn/browser-compat-data?
https://github.com/mdn/browser-compat-data/pull/8458/ has been merged for instance, but still not live as far as I can see

**Comment by peterbe** [comment]
> For example, there’s a certain time each day when the push to production happens, right? There’s some cron job that runs at least once a day, I guess? So maybe we can document what time that actually is.
> 
> And then along with that, we have a rough idea of how long the cache invalidations take — 4 hours, or 6 hours, or 8 hours?
> 
> So maybe in the end we could say something like, _It should take at most 32 hours (or whatever the actual number works out to) for a change you contribute to finally show up on the production site_.

The complete build runs every 24h at around 7PM US/Eastern time. But we don't specify the time so I think we're at the mercy of GitHub's pick of the clock. 

The CDN is configured to hold on to things 24h (coincidence number) too. 
So the max ("worst case") time from merged-to-`main` till you can see it in the browser is 48h. 

If you can think up a good place to explain and document this in the README (or contribution guide) we can solidifiy it there.

**Comment by peterbe** [comment]
> Is it the same for mdn/browser-compat-data?
> [mdn/browser-compat-data#8458](https://github.com/mdn/browser-compat-data/pull/8458) has been merged for instance, but still not live as far as I can see

That project makes a release (to npmjs.com) roughly every 7 days. I think it's almost always on Thursdays. 
Once it's released, Dependabot will upgrade our version of it in mdn/yari and it should get auto-merged. Then, in the next 24h it will be used in a new build.

**Comment by sideshowbarker** [comment]
> If you can think up a good place to explain and document this in the README (or contribution guide) we can solidifiy it there.

Raised https://github.com/mdn/content/pull/959

**Pull request merged**